content/docs/400-guides/400-observability/400-telemetry/200-metrics.mdx

```diff
@@ -427,3 +427,39 @@ FrequencyState {
 }
 */
 ```
+
+## Tagging Metrics
+When creating metrics, you can add tags to them. Tags are key-value pairs that provide additional context, helping in categorizing and filtering metrics. This makes it easier to analyze and monitor specific aspects of your application.
+
+
+### Tagging multiple Metrics
+
+Use `Effect.tagMetrics` to apply tags to all metrics created in the same context. This is useful for adding common tags that apply to multiple metrics.
+
+```ts twoslash
+import { Metric, Effect } from "effect"
+ 
+const taskCount = Metric.counter("task_count")
+const task1 = Effect.succeed(1).pipe(Effect.delay("100 millis"))
```

Comment:
Maybe this can just be Effect.sleep("100 millis")?

Reply:
it doesn't type check:

```ts
Argument of type 'Effect<void, never, never>' is not assignable to parameter of type 'Effect<number, never, never>'
```